MobSF/settings.py

```diff
@@ -312,8 +312,8 @@
     # ==============================================
 
     # ================HTTPS PROXY ===============
+    PROXY_IP = '127.0.0.1'
     PROXY_PORT = 1337  # Proxy Port
-    ROOT_CA = '0025aabb.0'
```

Comment:
If we don't have a programatic cross platform way to generate this from PEM file without depending on 3rd party installation locations, Let's hardcode this value. To be honest, we should generate the cert name only if the hardcoded certificate name is not respected.

Reply:
Can be removed if the above change is accepted.